src/index.html

```diff
@@ -3,11 +3,287 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&family=Space+Grotesk:wght@700&family=Space+Mono&display=swap" rel="stylesheet">
+    <title>B&O</title>
+    <link rel="stylesheet" href="./main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <navigation class="navigation header__navigation">
```

Comment:
HTML5 does not have a 'navigation' element. You might have meant to use 'nav' instead of 'navigation'.